src/py42/clients/securitydata.py

```diff
@@ -470,6 +486,12 @@ def node_guid(self):
         return self._node_guid
 
 
+def _get_first_matching_version(versions, md5_hash):
+    exact_match = next((x for x in versions if x[u"fileMD5"] == md5_hash), None,)
```

Comment:
there are certain cases where a trailing comma in the tuple like this can cause our formatter to throw odd errors (specifically, `flake8`). It isn't now (obviously), but the comma is not needed and I am scarred from my past.

Reply:
Ran into same issue of the two fighting and failing. Had to manually remove the trailing comma after the 3.6+ args was added.